lib/SILOptimizer/Utils/Devirtualize.cpp

```diff
@@ -515,10 +515,8 @@ bool swift::canDevirtualizeClassMethod(FullApplySite AI,
   }
 
   // We need to disable the  “effectively final” opt if a function is inlinable
-  if (isEffectivelyFinalMethod &&
-      ((F->getResilienceExpansion() == ResilienceExpansion::Minimal) ||
-       AI.getFunction()->getResilienceExpansion() ==
-           ResilienceExpansion::Minimal)) {
+  if (isEffectivelyFinalMethod && AI.getFunction()->getResilienceExpansion() ==
+                                      ResilienceExpansion::Minimal) {
     DEBUG(llvm::dbgs() << "        FAIL: Could not optimize function because "
                           "it is an effectively-final inlinable: "
                        << F->getName() << "\n");
```

Comment:
This should probably be using `AI.getFunction()` too, right?

Reply:
Right!